config.yml

```diff
@@ -28,13 +28,13 @@ Wiki:
     # Each wiki category is followed by a list of nodes in the inheritance tree whose child objects
     # should be in that category. Objects may also be explicitly specified.
     # Order matters since later, more specific entries can override earlier, more general ones.
```

Comment:
The first things in the row are the wiki categories, the things between [] are what goes into those categories